packages/import/src/importers/trelloJson/TrelloJsonImporter.ts

```diff
@@ -155,12 +155,13 @@ export class TrelloJsonImporter implements Importer {
         archived: card.closed || cardList?.closed,
       });
 
-      const allLabels = card.labels?.map(label => ({
-        id: label.id,
-        color: mapLabelColor(label.color),
-        // Trello allows labels with no name and only a color value, but we must specify a name
-        name: label.name || `Trello-${label.color}`,
-      })) ?? [];
+      const allLabels =
+        card.labels?.map(label => ({
+          id: label.id,
+          color: mapLabelColor(label.color),
+          // Trello allows labels with no name and only a color value, but we must specify a name
+          name: label.name || `Trello-${label.color}`,
+        })) ?? [];
```

Comment:
Is this from #687 ?

Reply:
it's just a line wrap that my formatter made, can revert but assumed it used the global formatting settings?